search/factory.py

```diff
@@ -80,7 +80,7 @@ def create_classic_api_web_app() -> Flask:
 
     app = Flask('search')
     app.json_encoder = ISO8601JSONEncoder
-    app.config.from_pyfile('config.py')   # type: ignore
+    app.config.from_pyfile('config.py') # type: ignore
```

Comment:
Please review [PEP8 re inline comments](https://www.python.org/dev/peps/pep-0008/#inline-comments)

```suggestion
    app.config.from_pyfile('config.py')  # type: ignore
```